plugins/star-rating/frontend/public/templates/feedback-popup.html

```diff
@@ -1,291 +1,408 @@
+            <div class="buttons-area-on-success">
+            </div>
+            <div class="modal-footer">
+                <a href="https://count.ly" target="_new">
+                    <img id="powered-by-countly" src="./star-rating/images/star-rating/powered-by-countly.svg">
+                    </img>
+                </a>
+            </div>
+        </div>
+        <script src="/star-rating/javascripts/tippy.all.min.js">
+        </script>
+        <script src="/javascripts/dom/jquery/jquery-1.8.3.min.js">
+        </script>
+        <script>
+            var Countly = Countly || {};
+            Countly.url = window.location.href.split("/")[0] + "//" + window.location.href.split("/")[2];
+            Countly.onload = Countly.onload || [];
 
-  (function() {
-    var cly = document.createElement('script'); cly.type = 'text/javascript'; 
-    cly.async = true;
-    // Enter url of script here (see below for other option)
-    cly.src = '/javascripts/min/countly.min.js';
-    cly.onload = function(){Countly.init()};
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(cly, s);
-  })();
-  
-  Countly.onload.push(function() {
-    /*
-    Helper methods which we need 
-    */
-    function hasClass(ele,cls) {
-      return !!ele.className.match(new RegExp('(\\s|^)'+cls+'(\\s|$)'));
-    }
-    function addClass(ele,cls) {
-      if (!hasClass(ele,cls)) ele.className += " "+cls;
-    }
-    function removeClass(ele,cls) {
-      if (hasClass(ele,cls)) {
-        var reg = new RegExp('(\\s|^)'+cls+'(\\s|$)');
-        ele.className=ele.className.replace(reg,' ');
-      }
-    }
-    function asyncForeach(array, fn, atEnd) {
-      var at = -1;
-      function next(shouldBreak) {
-        if (shouldBreak || ++at == array.length) {
-          if (atEnd) {
-            setTimeout(atEnd);
-          }
-        } else {
-          setTimeout(fn, 0, array[at], next);
-        }
-      }
-      next();
-    }
-    function isValidEmail(email) { 
-      return /^.+@.+\..+$/.test(email); 
-    }
-    function getParameterByName(name, url) {
-      if (!url) url = window.location.href;
-      name = name.replace(/[\[\]]/g, "\\$&");
-      var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
-          results = regex.exec(url);
-      if (!results) return null;
-      if (!results[2]) return '';
-      return decodeURIComponent(results[2].replace(/\+/g, " "));
-    }
+            (function() {
+                var cly = document.createElement('script');
+                cly.type = 'text/javascript';
+                cly.async = true;
+                // Enter url of script here (see below for other option)
+                cly.src = '/javascripts/min/countly.min.js';
+                cly.onload = function() {
+                    Countly.init()
+                };
+                var s = document.getElementsByTagName('script')[0];
+                s.parentNode.insertBefore(cly, s);
+            })();
```

Comment:
I think we need to exclude Web SDK from here completely, it will not track what we want it to track eitherway.